analyzer/src/main/kotlin/managers/utils/PythonInspector.kt

```diff
@@ -112,10 +116,19 @@ internal object PythonInspector : CommandLineTool, Logging {
     @Serializable
     internal data class Result(
         @SerialName("files") val projects: List<Project>,
-        @SerialName("resolved_dependencies_graph") val resolvedDependenciesGraph: List<ResolvedDependency>,
+        @SerialName("resolved_dependencies_graph")
+        @Serializable(with = DependencyGraphDeserializer::class)
```

Comment:
Would it make sense to do the changes in this file in a separate commit that can be cleanly reverted once the issue is fixed, but the changes to `PipFunTest` are kept?